beta/src/pages/learn/javascript-in-jsx-with-curly-braces.md

```diff
@@ -1,25 +1,24 @@
 ---
-title: JavaScript in JSX with Curly Braces
+title: घुंघराले ब्रेसिज़ के साथ JSX में JavaScript
 ---
 
 <Intro>
 
-JSX lets you write HTML-like markup inside a JavaScript file, keeping rendering logic and content in the same place. Sometimes you will want to add a little JavaScript logic or reference a dynamic property inside that markup. In this situation, you can use curly braces in your JSX to open a window to JavaScript.
-
+JSX आपको तर्क और सामग्री को एक ही स्थान पर प्रस्तुत करते हुए, JavaScript फ़ाइल के अंदर HTML जैसा मार्कअप लिखने देता है। कभी-कभी आप थोड़ा JavaScript तर्क जोड़ना चाहेंगे या उस मार्कअप के अंदर एक गतिशील संपत्ति का संदर्भ देना चाहेंगे। इस स्थिति में, आप JavaScript के लिए एक विंडो खोलने के लिए अपने JSX में घुंघराले ब्रेसिज़ का उपयोग कर सकते हैं।
```

Comment:
`तर्क` => `rendering लॉजिक`

I can't think of anything better. let me know if you have any other suggestions?

Reply:
अनुवाद

Reply:
I don't think `अनुवाद` would convey the same meaning as logic with respect to react.

Lets keep it `rendering लॉजिक` for now. may be we can revisit this later

Reply:
@lucifertrj Here you missed `rendering` word

we should make it `rendering लॉजिक`